### ISSUE TITLE
QF_BV instance segment fault (NPD) 

### ISSUE THREAD

**Issue by rainoftime** [CVC4/CVC4#3913]
Hi,
For this formula,

[cvcnpd.txt](https://github.com/CVC4/CVC4/files/4285747/cvcnpd.txt)

cvc4-nightly (http://cvc4.cs.stanford.edu/downloads/builds/x86_64-linux-opt/unstable/cvc4-2020-03-03-x86_64-linux-opt) incremental mode
crashed

~~~~
CVC4 suffered a segfault.
Offending address is 0x29
Looks like a NULL pointer was dereferenced.
~~~~

OS: Ubuntu 16.04

**Comment by ajreynol** [comment]
This is also due to `--solve-real-as-int`, it is fixed in #3918 .

**Comment by rainoftime** [comment]
In the new version http://cvc4.cs.stanford.edu/downloads/builds/x86_64-linux-opt/unstable/cvc4-2020-03-07-x86_64-linux-opt

`--solve-real-as-int` causes a NPD for the following formula 
[realasint.txt](https://github.com/CVC4/CVC4/files/4301885/realasint.txt)
